book/types_of_data.md

```diff
@@ -42,7 +42,7 @@ A string of characters that represents text. There are a few ways these can be c
   - `$"6 x 7 = (6 * 7)"`
   - `ls | each { |it| $"($it.name) is ($it.size)" }`
 - Bare strings
-  - `hello`
+  - `"hello"`
```

Comment:
I think under the title bare strings we should give an example for something like before `print hello` or `git branch hello`

Reply:
Ah, quite right.